store/sqlstore/user_store.go

```diff
@@ -1347,23 +1347,20 @@ func (us SqlUserStore) GetProfilesNotInTeam(teamId string, offset int, limit int
 
 func (us SqlUserStore) GetEtagForProfilesNotInTeam(teamId string) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		updateAt, err := us.GetReplica().SelectInt(`
-            SELECT
-                u.UpdateAt
-            FROM Users u
-            LEFT JOIN TeamMembers tm
-                ON tm.UserId = u.Id
-                AND tm.TeamId = :TeamId
-                AND tm.DeleteAt = 0
-            WHERE tm.UserId IS NULL
-            ORDER BY u.UpdateAt DESC
-            LIMIT 1
-            `, map[string]interface{}{"TeamId": teamId})
 
+		var querystr string
+		querystr = `
+			SELECT 
+				CONCAT(MAX(UpdateAt), '.',  COUNT(Id)) as etag
+			FROM Users as u
+				LEFT JOIN TeamMembers tm ON tm.UserId=u.Id AND tm.TeamId=:TeamId AND tm.DeleteAt=0
```

Comment:
From a code review perspective, can you adopt the SQL styling we're using elsewhere, basically putting each SQL statement on its own line, aligned with the previous lines, and with spaces around the `=` characters?